lib/ScriptFileStorage.js

```diff
@@ -149,17 +150,20 @@ $class.listScripts = function(rootFolder, pattern, callback) {
   //    callback
   // );
 
+  debug('glob %s on %s', pattern, rootFolder);
+
   glob(
     pattern,
     {
       cwd: rootFolder,
+      follow: true,
+      realpath: true,
       strict: false
     },
     function(err, result) {
       if (result) {
-        result = result.map(function(relativeUnixPath) {
-          var relativePath = relativeUnixPath.split('/').join(path.sep);
-          return path.join(rootFolder, relativePath);
+        result = result.map(function(unixPath) {
+          return unixPath.split('/').join(path.sep);
```

Comment:
yes - i had to fix the code above to avoid prepending the rootfolder twice (which breaks inspector). i also had a little exchange with isaacs about the realpath returned paths being absolute now (and the match results not). https://github.com/isaacs/node-glob/issues/184